src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -68,7 +69,7 @@ public function resolve(Request $request, ArgumentMetadata $argument): array
         } elseif (false === $object = $this->find($manager, $request, $options, $argument)) {
             // find by criteria
             if (!$criteria = $this->getCriteria($request, $options, $manager, $argument)) {
-                return [];
+                throw new NearMissValueResolverException(sprintf('Cannot find mapping for "%s": declare the identifier using either the #[MapEntity] attribute or mapped route parameters.', $options->class));
```

Comment:
not only identifiers, right? (this applies to the deprecation also, isn't it?)
```suggestion
                throw new NearMissValueResolverException(sprintf('Cannot find mapping for "%s": declare it using either the #[MapEntity] attribute or mapped route parameters.', $options->class));
```

Reply:
I assumed ”identifier” meant “whatever data identifying an entity” in this context ¯\\_(ツ)\_/¯

Updating.